src/device-registry/app.js

```diff
@@ -16,61 +16,6 @@ const routes = require("@routes");
 
 mongodb;
 
-const { Kafka } = require("kafkajs");
-const kafka = new Kafka({
```

Comment:
the KAFKA consumption logic removed from the app.js file and moved to the WWW start script. cc: @BenjaminSsempala